### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Tomekmularczyk**

### CODE REVIEW

File: src/Contact.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  handleSubmit = e => {
+    postData(new FormData(e.target), e => {
+      this.setState({ isSubmited: true });
+    });
+  };
+
+  render() {
+    // console.log(this.state);
```
Comment (betafcc) [inline src/Contact.js:25]: Clean useless comment

File: src/Contact.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    return (
+      <div className="container">
+        <img
+          className="logo"
+          src="https://www.netguru.com/hubfs/images/landing_pages/og72x.png"
```
Comment (blueturtle13g) [inline src/Contact.js:31]: use constants for such strings

File: src/Contact.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        />
+        <div className="header">Contact</div>
```
Comment (blueturtle13g) [inline src/Contact.js:33]: use semantic html, not div for header

File: src/Contact.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <div className="title">Use form below to contact us:</div>
```
Comment (blueturtle13g) [inline src/Contact.js:34]: don't use div for strings, instead use h tags

File: src/App.js [hunk 5]
```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import Contact from "./Contact";
 
 function App() {
```
Comment (blueturtle13g) [inline src/App.js:4]: use arrow functions instead

**Review by blueturtle13g — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**